react-app-rewired/headers/csps/fiatRamps/onRamper.ts

```diff
@@ -0,0 +1,6 @@
+import type { Csp } from '../../types'
+
+export const csp: Csp = {
+  'frame-src': ['https://widget.onramper.com', 'https://buy.moonpay.com'],
+  'frame-ancestors': ['*']
```

Comment:
Discussed this one out, ideally `self` would be safer but given CSPs are isolated to their specific context, this is fine